pkg/blockchain/nonceManager.go

```diff
@@ -51,7 +52,7 @@ func NewSQLBackedNonceManager(db *sql.DB, logger *zap.Logger) *SQLBackedNonceMan
 	return &SQLBackedNonceManager{
 		db:      db,
 		logger:  logger.Named("SQLBackedNonceManager"),
-		limiter: NewOpenConnectionsLimiter(BestGuessConcurrency),
+		limiter: NewOpenConnectionsLimiter(MaxConcurrentRequests),
```

Comment:
@mkysel is this change a bad idea?

Reply:
you will break the deployed environment. 2x64 is too much if you hammer in HA. But it is worth exploring. I have not done any tuning.

Reply:
OK. I'll revert it for now and play around a bit